app/code/Magento/Cron/Model/Schedule.php

```diff
@@ -241,4 +256,15 @@ public function tryLockJob()
         }
         return false;
     }
+
+    /**
+     * @deprecated
+     * @return \Magento\Framework\Stdlib\DateTime\TimezoneInterface
+     */
+    private function getDateTime()
+    {
+        $this->timeZone = $this->timeZone ?: \Magento\Framework\App\ObjectManager::getInstance()
```

Comment:
Static access to the object manager is highly discouraged. The interface primarily exists for the unserialization use case. It should not be used outside of that. The object manager will not be necessary if to inject the dependency as required on constructor. See the constructor comment above.